src/SFA.DAS.AssessorService.Application/Handlers/EmailHandlers/SendEmailHandler.cs

```diff
@@ -18,60 +19,39 @@ public class SendEmailHandler : IRequestHandler<SendEmailRequest>
         private const string SystemId = "AssessorService";
         private const string ReplyToAddress = "digital.apprenticeship.service@notifications.service.gov.uk";
         private const string Subject = "EPAO user to approve";
-        private readonly INotificationsApi _notificationsApi;
-        private readonly IEMailTemplateQueryRepository _eMailTemplateQueryRepository;
+        private readonly INotificationsApi _notificationsApi;        
         private readonly ILogger<SendEmailHandler> _logger;
 
-        public SendEmailHandler(INotificationsApi notificationsApi, IEMailTemplateQueryRepository eMailTemplateQueryRepository, ILogger<SendEmailHandler> logger)
+        public SendEmailHandler(INotificationsApi notificationsApi, ILogger<SendEmailHandler> logger)
         {
-            _notificationsApi = notificationsApi;
-            _eMailTemplateQueryRepository = eMailTemplateQueryRepository;
+            _notificationsApi = notificationsApi;            
             _logger = logger;
         }
 
       
         public async Task<Unit> Handle(SendEmailRequest message, CancellationToken cancellationToken)
         {
-            var emailTemplate = message.EmailTemplate;
+            var emailTemplateSummary = message.EmailTemplateSummary;            
+            var personalisationTokens = GetPersonalisationTokens(message.Tokens);
 
-            if (emailTemplate != null && !string.IsNullOrWhiteSpace(message.Email))
+            if (emailTemplateSummary != null && !string.IsNullOrWhiteSpace(message.Email))
+            {                
+                await SendEmailViaNotificationsApi(message.Email, emailTemplateSummary.TemplateId, emailTemplateSummary.TemplateName, personalisationTokens);
+            }
+            else if (emailTemplateSummary != null && emailTemplateSummary.Recipients != string.Empty)
             {
-                var personalisationTokens = GetPersonalisationTokens(message.Tokens);
-                await SendEmailViaNotificationsApi(message.Email, emailTemplate.TemplateId, emailTemplate.TemplateName, personalisationTokens);
-
-                if(emailTemplate.RecipientTemplate != null)
-                    await SendEmailToRecipients(emailTemplate.RecipientTemplate, personalisationTokens);
-
+                await SendEmail(emailTemplateSummary, personalisationTokens);
```

Comment:
SendEmailToTemplateRecipients may make this clearer as it has some pluralisation in the name